src/utils/error-to-string.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function errorToString(err) {
```

Comment:
@matteofigus I wasn't sure where to put this but but it would just see if it has a message attribute that is a string and returns that.